examples/file-and-cloudinary-upload/app/routes/cloudinary-upload.tsx

```diff
@@ -42,20 +44,28 @@ export const action: ActionFunction = async ({ request }) => {
 
 export default function Index() {
   const data = useActionData<ActionData>();
+
   return (
     <>
       <Form method="post" encType="multipart/form-data">
         <label htmlFor="img-field">Image to upload</label>
+
```

Comment:
Let's just remove all of the opinionated formatting changes from the PR if you don't mind, ESLint should be hands-off here 🙏